helix-view/src/editor.rs

```diff
@@ -274,6 +274,8 @@ pub struct Config {
     /// Whether to color modes with different colors. Defaults to `false`.
     pub color_modes: bool,
     pub soft_wrap: SoftWrap,
+    /// Whether to center views by default, Defaults to `false`.
+    pub centered_views: bool,
```

Comment:
I'd remove the config option since it seems unlikely someone would want this always on.